sdcm/tester.py

```diff
@@ -1629,18 +1627,8 @@ def list2reason(exc_list):
         else:  # Python 3.2 - 3.3 or 3.0 - 3.1 and 2.7
             result = getattr(self, '_outcomeForDoCleanups', self._resultForDoCleanups)  # pylint: disable=no-member
         error = list2reason(result.errors)
-        failure = list2reason(result.failures)
-        ok = not error and not failure
-
-        if not ok:
-            typ, text = ('ERROR', error) if error else ('FAIL', failure)
-            TEST_LOG.error("=" * 70)
```

Comment:
why did you remove this prints ?  since the event is printed ?

Reply:
yes, I print the event